doc/articles/create-an-app-rider.md

```diff
@@ -9,6 +9,9 @@ uid: Uno.GettingStarted.CreateAnApp.Rider
 
 ## Create the App
 
+> [!IMPORTANT]
+> Current versions of Rider (2024.1 and earlier) do not support creating Uno Platform projects using the "New Solution" dialog, even if the Uno Platform project template appears. Make sure to follow the directions below.
```

Comment:
@jeromelaban this sentence would need to change after @Takoooooo current changes for the template right?

Reply:
yes